code/game/objects/items/weapons/material/material_armor.dm

```diff
@@ -190,7 +195,12 @@ Protectiveness | Armor %
 
 		if(!isnull(material.conductivity))
 			siemens_coefficient = between(0, material.conductivity / 10, 10)
-		slowdown = between(0, round(material.weight / 10, 0.1), 6)
+
+		var/slowdownModified = between(0, round(material.weight / 10, 0.1), 6)
+
+		var/slowdownUncapped = (material_slowdown_multiplier * slowdownModified) - material_slowdown_modifier
```

Comment:
Quibble: you declare this var and store a value that you only use once, on the next line. You could just put the calc into it instead.

Reply:
I got asked to seperate this out for readability.